domain/domain.go

```diff
@@ -826,25 +826,36 @@ func (do *Domain) LoadBindInfoLoop(ctx sessionctx.Context) error {
 		return err
 	}
 
-	do.loadBindInfoLoop()
+	do.globalBindHandleWorkerLoop()
 	do.handleInvalidBindTaskLoop()
 	return nil
 }
 
-func (do *Domain) loadBindInfoLoop() {
+func (do *Domain) globalBindHandleWorkerLoop() {
 	do.wg.Add(1)
 	go func() {
 		defer do.wg.Done()
-		defer recoverInDomain("loadBindInfoLoop", false)
+		defer recoverInDomain("globalBindHandleWorkerLoop", false)
+		loadTicker := time.NewTicker(bindinfo.Lease)
+		captureBaselineTicker := time.NewTicker(bindinfo.Lease)
```

Comment:
Can we just use a single ticker for 2 actions?

Reply:
Oh, we can.